tests/ut/test_iterator.cc

```diff
@@ -111,9 +127,10 @@ TEST_CASE("Test Iterator Mem Index With Float Vector", "[float metrics]") {
 
     SECTION("Test Search with Bitset using iterator") {
         using std::make_tuple;
-        auto [name, gen, threshold] = GENERATE_REF(table<std::string, std::function<knowhere::Json()>, float>({
-            make_tuple(knowhere::IndexEnum::INDEX_HNSW, hnsw_gen, hnswlib::kHnswSearchKnnBFFilterThreshold),
-        }));
+        auto [name, gen, threshold] = GENERATE_REF(table<std::string, std::function<knowhere::Json()>, float>(
```

Comment:
seems threshold is unused, pls help remove it, and `kBruteForceRecallThreshold` at the top of the file.

this was used in knn ut to ensure that at high filter rate we should see a higher recall since we switch to brute force. But for now in iterators we don't switch to BF based on filter rate, thus not necessary. We may add such strategy later but don't need it now.

Reply:
removed.